### ISSUE TITLE
aria-describedby="figcaption too much ...

### ISSUE THREAD

**Issue by sprungmarker** [richardshepherd/TwentyTenFive#12]
aria-describedby="figcaption is too much for screenreader usage.

So alternative text of the image is read or title, figcaption via aria and the caption - mostly the same text. I think aria-describedby is too much in this case.

Thanks.

**Comment by richardshepherd** [comment]
aria-described by is used for when images do have alt text. If there was no alt text you'd use aria-labelledby.
